spaceship/docs/iTunesConnect.md

```diff
@@ -231,29 +227,13 @@ build.train_version           # => "0.9.21" (the version number)
 build.install_count           # => 1
 build.crash_count             # => 0
 
-build.testing_status          # => "Internal" or "External" or "Expired" or "Inactive"
+build.internal_state         # => testflight.build.state.testing.ready
+build.external_state         # => testflight.build.state.submit.ready
 ```
 
-You can even submit a build for external beta review
+You can even submit a build for external beta review (after you have set all necessary metadata - see above)
 ```ruby
-parameters = {
-  changelog: "Awesome new features",
-  description: "Why would I want to provide that?",
-  feedback_email: "contact@company.com",
-  marketing_url: "http://marketing.com",
-  first_name: "Felix",
-  last_name: "Krause",
-  review_email: "contact@company.com",
-  phone_number: "0123456789",
-  significant_change: false,
```

Comment:
Are these options not available any more at all?

Reply:
AFAIK, the way to submit build for external review is through TestFlight now. All of the options, aside from significant change are over there @KrauseFx.

Reply:
The method submit_for_testflight_review from TestFlight::Build doesn't allow parameters. Correct me  if I'm wrong, but I noticed that submit_for_beta_review is not supported anymore.